README.md

```diff
@@ -27,6 +27,35 @@ the following within the repo:
 go run .
 ```
 
+## Conversion of Ingress resources to Gateway API
+
+### Processing Order and Conflicts
+
+Ingress resources will be processed with a defined order to ensure deterministic generated Gateway API configuration.
+This should also determine precedence order of Ingress resources and routes in case of conflicts.
+
+Ingress resources with the oldest creation timestamp will be sorted first and therefore given precedence.
+If creation timestamps are equal, then sorting will be done based on the namespace/name of the resources.
+If an Ingress rule conflicts with another (e.g. same path match but different backends) an error will be reported for the one that sorted later.
```

Comment:
While really like this process, I feel like I should mention that some users expect the ingress-nginx behavior of merging matching Ingress objects into load-balanced sets. I'm okay with not doing this for now, but we need to remember it for later.